modular_skyrat/modules/tribal_extended/code/weapons/sword.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/claymore/bone
+	name = "Bone Sword"
+	desc = "Jagged pieces of bone are tied to what looks like a goliaths femur."
+	icon = 'modular_skyrat/modules/tribal_extended/icons/items_and_weapons.dmi'
+	lefthand_file = 'modular_skyrat/modules/tribal_extended/icons/swords_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/tribal_extended/icons/swords_righthand.dmi'
+	icon_state = "bone_sword"
+	inhand_icon_state = "bone_sword"
+	slot_flags = ITEM_SLOT_BELT | ITEM_SLOT_BACK
+	force = 20
```

Comment:
Remember that the big force is also a factor in Ashwalker vs Miner scenarios. Not saying it's a bad thing, just a thing to keep in mind.

Reply:
Spear (22) and axe (23) got slightly higher numbers but one's got reach & requires two hands while the other requires two hands and has no reach, so really not sure what to think about this one! Seems fine to me.

Reply:
Ah. Was under the impression that spears dealt 18 damage.

Reply:
Yeah man, bone spears a hair stronger! Greatly appreciated too, lol. Just enough to handle airlocks.

Reply:
Not touching any code outside this modular file for the PR, swords have some nerfed AP a little